validator/validate.py

```diff
@@ -0,0 +1,80 @@
+"""Configuration files validator"""
+
+import os
+from pathlib import Path
+
+import jsonschema
+import yaml
+
+from validator.config.config import Configuration, SupportedValidations
+from validator.errors.errors import InvalidTargetPathError
+from validator.schemas.schema import SCHEMAS
+
+
+def build_app_path():
+    """Builds the path for /apps folder"""
+    target_path = Configuration.get_target_path()
+    if target_path is None:
+        raise InvalidTargetPathError(target_path)
+    return os.path.join(target_path, "apps")
+
+
+def build_project_conf_path():
+    """Builds the path for project.yaml"""
+    target_path = Configuration.get_target_path()
+    if target_path is None:
+        raise InvalidTargetPathError(target_path)
+    return os.path.join(target_path, "project.yaml")
```

Comment:
I think there is some similar function in `nestor-api/lib/config`

Reply:
If we plan to do it then standalone then we can keep these ones, no?